src/main/java/com/assignment/Assignment4.java

```diff
@@ -0,0 +1,8 @@
+package com.assignment;
+public class Assignment4 {
+    public static String replacement() {
+        String IOCL = "Indian Oil Corporation Ltd";
+        String Corp = IOCL.substring(IOCL.indexOf('C'),IOCL.lastIndexOf('n')+1);
+        return Corp;
+       }
+}
```

Comment:
Please find my below comments :

The name for the function is not a meaningful one.
Any reason why we we are defining this as a static function?
This is not an optimised approach. Lets create a generic function instead of hardcoding like :
`String getSubString(String originalString, String stringToGet)`